docs/en/advanced-concepts/settings.md

```diff
@@ -86,7 +86,9 @@ Settings are implemented via the following API endpoints:
 |`keyboardAutocorrection`| Changes the 'Auto-Correction' preference in _Keyboards_ setting. Defaults to `false` when WDA starts as xctest. | `true`, `false` |1.14.0+|
 |`keyboardPrediction`| Changes the 'Predictive' preference in _Keyboards_ setting. Defaults to `false` when WDA starts as xctest. | `true`, `false` |1.14.0+|
 |`snapshotTimeout` | Changes the accessibility snapshots resolution timeout. _Snapshots_ are mainly used for page source generation, XML lookup and attributes retrieval. It might be necessary to increase this value if the actual page source is very large and contains hundreds of UI elements. Defaults to 15 seconds. | e.g. `10`, `100` (seconds) |1.15.0+|
-|`snapshotMaxDepth`| Changes the number of maximum depth for traversing elements source tree. It may help to prevent out of memory while getting the source, but it might restrict the depth of source tree. Defaults to `50` | e.g. `100` | 1.17.0+ |
+|`snapshotMaxDepth`| Changes the value of maximum depth for traversing elements source tree. It may help to prevent out of memory while getting the source, but it might restrict the depth of source tree. Please consider to restrict this value if you observed an error like _Timed out snapshotting com.apple.testmanagerd..._ message or _Cannot get 'xml' source of the current application_ in your Appium log since they are possibly timeout related. A part of elements source tree might be lost if the value was small. Defaults to `50` | e.g. `100` | 1.17.0+ |
```

Comment:
..It may help to prevent out of memory or timeout errors while getting the page source..